manage.py

```diff
@@ -77,7 +77,6 @@ def init_db(force=False, create_db=False):
 
     click.echo("Database has been initialized successfully!")
 
-
```

Comment:
Hey, can you add this line back in ?
It's not a big deal, but having two empty lines before a function definition is the preferred formatting according PEP8